### PR TITLE
Update PR template

### LINKED ISSUE #4787: Update the all repo pull request templates and add system architecture change section
**What we're after:**
Update the openFEC pull request template (https://github.com/fecgov/openFEC/blob/develop/PULL_REQUEST_TEMPLATE.md) 
and other repo pull request templates (except PROXY repo) to include changes that may be needed if system architecture changes are being made. Add this as a new section to the PR template. 

Repos to include:
- openFEC
- fec-eregs
- fec-cms

**Action items:**
- [x] Bring all PR templates in line (exclude PROXY repo)
- [x] Add system architecture change section to updated PR template
- [x] Templates should be clear and simple

**Completion criteria:**
- [ ] All templates are all similar in structure and include new system architecture section

### PR DESCRIPTION
**Pull request by patphongs**
## Summary

- Resolves #4787 

_Adds github PR template per [reviewed drafted document](https://docs.google.com/document/d/1fBMFNUr6GFJCHwhHNOP-VTIcZ3CJR44sWUQxjkahoVY/edit#) 🔒._

### Required reviewers

One designer and one developer reviews are required prior to merge.

## Impacted areas of the application
List general components of the application that this PR will affect:

-  Only github PR templates

## Screenshots

Screen Shot 2021-03-22 at 9 40 35 AM

## Related PRs

branch | PR
------ | ------
feature/4787-pr-template-changes (fec-cms) | [link](https://github.com/fecgov/fec-cms/pull/4489)
feature/4787-pr-template-changes (fec-eregs) | [link](https://github.com/fecgov/fec-eregs/pull/576)

## How to test
- You can view the screenshot above 
 
--OR--

- Copy the text from the PULL_REQUEST_TEMPLATE.md file
- Create a new issue in any repository and paste what was copied in the PULL_REQUEST_TEMPLATE.md file to the body of the new issue
- Click on the "Preview" tab to see what the pull request template will look like

____

### CODE REVIEW

**Comment by JonellaCulmer** [comment]
@patphongs It looks like it to me, but wanted to confirm that all three repos have the exact same template? Wasn't sure if I was missing something from your screenshots.

**Comment by patphongs** [comment]
> It looks like it to me, but wanted to confirm that all three repos have the exact same template? Wasn't sure if I was missing something from your screenshots.

@JonellaCulmer That's correct, all 3 have the exact same template. The ticket specified that we want to make sure that all 3 repos (fec-cms, fec-eregs, and openfec) are using the same template.

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -22,3 +24,14 @@ branch | PR
 ------ | ------
 fix/other_pr | [link]()
 feature/other_pr | [link]()
+
+## How to test
+Include any information that may be helpful to the reviewer(s).
+_This might include:_
+
+- links to sample pages to test
+- Any local environmental setup that is unusual (env vars, API version to point to, etc)
```
Comment (JonellaCulmer) [inline PULL_REQUEST_TEMPLATE.md:33]: Can we say here "...setup, **including the** unusual (env vars..."

**Review by JonellaCulmer — COMMENTED** [review]
Minor suggested tweak to the "How to test" section.

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (patphongs) [reply PULL_REQUEST_TEMPLATE.md]: Thanks @JonellaCulmer, I made the change to each of the PR templates for openfec, fec-cms, and fec-eregs

**Review by JonellaCulmer — APPROVED** [review]
Thanks for making that one change. Looks good to me.

**Comment by patphongs** [comment]
Closing in favor of this more consolidated PR: https://github.com/fecgov/openFEC/pull/4814

**Pull request closed without merging**